lib/facilities/gis_metadata_configuration.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/facility_parser'
+
+module Facilities
+  # Configuration class used to setup the environment used by client
+  class GisMetadataConfiguration < Common::Client::Configuration::REST
+    def base_path
+      Settings.locators.gis_base_path
+    end
+
+    def service_name
+      'FL'
+    end
+
+    def connection
+      Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
+        conn.use :breakers
+        conn.request :json
+
+        # Uncomment this if you want curl command equivalent or response output to log
```

Comment:
Seems like all of this should be removed instead of commented out. Or toggled?

Reply:
Could you explain "toggle"? Do you mean the activerecord method?

Reply:
@edmkitty no -- I mean you've got commented out code here that ... might need to be used at some point? Wondering if there's either some kind of toggle value (environment, log level, etc) that would indicate this output should be injected. Or more likely this is something that was useful during dev but can now be removed